test/modules/TestResultCoordinator/ITestResultReport.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    /// <summary>
+    /// This defines the basic properties of a test result report.
+    /// </summary>
+    interface ITestResultReport
+    {
+        string TrackingId { get; }
+
+        string Title { get; }
+
+        string ResultType { get; }
+
+        string ExpectSource { get; }
```

Comment:
>ExpectSource [](start = 15, length = 12)

nit: just Expected and Actual

Reply:
This is the expected source name e.g. "loadGen1.Send" or "Relayer1.receive".  Prefer to keep "Source".